EIPS/eip-00.md

```diff
@@ -464,6 +478,20 @@ A builder that chooses not to enforce the rules from EIP-9999 **must** take care
 against the mid-block state at the position where it is being included into a block.
 Otherwise, the resulting block is likely to end up being invalid.
 
+### Block structure diagram
+
+Here is a visual representation of a block that contains multiple Account Abstraction Transactions.
+The validation parts of AA transactions are executed in a separate position in a block, but are not considered
+to be separate a transaction:
```

Comment:
Would it be clearer if we write "The validation parts of AA transactions are executed as separate transactions, but are not represented as separate transactions in the block data." ?